tools/code_format/check_format.py

```diff
@@ -773,6 +773,16 @@ def checkSourceLine(line, file_path, reportError):
                 "Lua API (bad light userdata pointer) on ARM64 architecture. See " +
                 "https://github.com/LuaJIT/LuaJIT/issues/450#issuecomment-433659873 for details.")
 
+  #Fix for https://github.com/envoyproxy/envoy/issues/10535
+  if file_path.endswith(PROTO_SUFFIX):
+    exclude_path = ['v1', 'v2', 'generated_api_shadow']
+    result = PROTO_VALIDATION_STRING.search(line)
+    if result is not None:
+      if not any(x in file_path for x in exclude_path):
+        reportError(
+            "Proto validation Error in file: %s. 'min_bytes' is DEPRECATED, Use 'min_len'." %
```

Comment:
I'd drop everything in this string up to the start of `'min_bytes'`, since it's redundant with other reports. See what `reportError` looks like in other uses in this function.

Reply:
thanks for comments. 
do you want this should be like: 
reportError(
              "Proto validation Error in file: %s. ." %  file_path )

Reply:
Done as per your suggestions. Thanks for quick review